### PR TITLE
Design single call page

### PR DESCRIPTION
**Pull request by markoT97**
Added react-bootstrap in package.json dependencies and using it created component SingleCallsView with all required handlers.

### CODE REVIEW

File: src/calls/components/SingleCallsView.jsx [hunk 1]
```diff
@@ -1,12 +1,546 @@
...
+        this.setState({ lastSuicidTry: event.target.textContent });
+    };
+
+    handleChangeShortContent = event => {
+        this.setState({ shortContent: event.target.value });
+    };
+
+    handleChangeNote = event => {
+        this.setState({ note: event.target.value });
+    };
```
Comment (coka) [inline src/calls/components/SingleCallsView.jsx:123]: It's possible to handle all of these input changes with one function, avoiding repetition. [This](https://reactjs.org/docs/forms.html#handling-multiple-inputs) is the relevant bit in React's documentation.

Another way would be to encapsulate input props and their change handlers via hooks. Have a look at [this conference talk](https://www.youtube.com/watch?v=dpw9EHDh2bM) as an introduction to hooks. Dan's section about input fields starts at 17:40.

You don't _have_ to refactor this now if the form works, but the refactor should happen at some point in the near future.

**Review by coka — CHANGES_REQUESTED** [review]
@markoT97 Please pull the latest `master` and rebase in order to fix the conflicts in `package.json`.

File: src/calls/components/SingleCallsView.jsx (hunk 1 above)
Comment (markoT97) [reply src/calls/components/SingleCallsView.jsx]: Ok, I will try to figure it out later. I fixed package.json

Comment (markoT97) [reply src/calls/components/SingleCallsView.jsx:123]: Ok, I figured out a logic, 
I will surely fix it in the future.

**Pull request merged**